docs/src/main/sphinx/connector/bigquery.md

```diff
@@ -439,6 +439,16 @@ the following features:
 ```{include} sql-delete-limitation.fragment
 ```
 
+### Wildcard tables
+
+The connector provides support to query multiple tables using a concise wildcard
+table notation.
```

Comment:
Worth noting from https://cloud.google.com/bigquery/docs/querying-wildcard-tables

> Each row in the wildcard table contains a special column, `_TABLE_SUFFIX`, which contains the value matched by the wildcard character.

Maybe it is worth exposing this hidden column for this specific use case in a follow-up (if this functionality has any traction).
cc @ebyhr